Source/WebCore/html/InputType.cpp

```diff
@@ -143,26 +149,35 @@ static InputTypeFactoryMap createInputTypeFactoryMap()
 
     InputTypeFactoryMap map;
     for (auto& inputType : inputTypes)
-        map.add(inputType.nameFunction(), std::make_pair(inputType.conditionalFunction, inputType.factoryFunction));
+        map.add(inputType.nameFunction(), InputTypeFactory { inputType.conditionalFunction, inputType.factoryFunction });
     return map;
 }
 
-static inline std::pair<InputTypeConditionalFunction, InputTypeFactoryFunction> findFactory(const AtomString& typeName)
+static inline std::pair<const AtomString&, InputTypeFactory*> findFactory(const AtomString& typeName)
 {
     static NeverDestroyed factoryMap = createInputTypeFactoryMap();
-    auto factory = factoryMap.get().get(typeName);
-    if (UNLIKELY(!factory.second))
-        factory = factoryMap.get().get(typeName.convertToASCIILowercase());
-    return factory;
+    auto& map = factoryMap.get();
```

Comment:
Not a change you should make, but I am curious where the cutoff for a map like this is where HashMap<AtomString, ...> starts winning over SortedArrayMap.